wolox-books/src/app/auth.guard.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, Router } from '@angular/router';
+import { SessionService } from './session.service';
+
+@Injectable({
+  providedIn: 'root'
+})
```

Comment:
```suggestion
@Injectable()
```
You already are provided in a module.